packages/emails/templates/attendee-location-change-email.ts

```diff
@@ -1,12 +1,19 @@
 import { renderEmail } from "../";
+import generateIcsString, { BookingAction } from "../lib/generateIcsString";
 import AttendeeScheduledEmail from "./attendee-scheduled-email";
 
 export default class AttendeeLocationChangeEmail extends AttendeeScheduledEmail {
   protected getNodeMailerPayload(): Record<string, unknown> {
     return {
       icalEvent: {
         filename: "event.ics",
-        content: this.getiCalEventAsString(),
+        content: generateIcsString({
```

Comment:
With the current logic of `generateIcsString`, it's adding similar logic as in it's caller to derive title, subtitle, and status.
e.g. 


How about we change `generateIcsString` signature to accept a `title`, `subtitle` and `status` as well. 

That way that logic stays together in the caller. `generateIcsString` wouldn't need to do switches over bookingAction and would be simpler.

Reply:
Implemented this feedback and updated tests in 63936a0b4726b38acbcb4d46d2045b3f937f11cf and 0c91dc1e0ae07a0c4d274212b9092cdfd3a6d6b4